hedera-mirror-rosetta/app/services/network/network_service_test.go

```diff
@@ -144,10 +145,30 @@ func TestNetworkOptions(t *testing.T) {
 	assert.Equal(t, expectedResult.Allow.HistoricalBalanceLookup, res.Allow.HistoricalBalanceLookup)
 	assert.Equal(t, expectedResult.Allow.OperationStatuses, res.Allow.OperationStatuses)
 	assert.Equal(t, expectedResult.Allow.OperationTypes, res.Allow.OperationTypes)
-	assert.Equal(t, len(expectedResult.Allow.Errors), len(res.Allow.Errors))
+	assert.True(t, compareErrors(expectedResult.Allow.Errors, res.Allow.Errors))
 	assert.Nil(t, e)
 }
 
+func compareErrors(e1, e2 []*rTypes.Error) bool {
```

Comment:
Don't we have similar logic in the `maphelper_test.go`? I think that we could add `comparator` package in the `tools` folder and have a generic `CompareUnorderedArrays` function and use it in places in the tests